plugins/destination/mongodb/client/migrate.go

```diff
@@ -2,11 +2,115 @@ package client
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/specs"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
-// MongoDB. No migrations needed :)
-func (*Client) Migrate(context.Context, schema.Tables) error {
+func (c *Client) Migrate(ctx context.Context, tables schema.Tables) error {
+	for _, t := range tables {
+		if err := c.migrateTable(ctx, t); err != nil {
```

Comment:
it's not that slow as-is, and errgroup would also introduce new variables like what's the concurrency should be and so on. also I'm not sure how it would behave if there's already significant amount of data in a collection and we're doing an index update, so I prefer to do it serially for now until some optimization is required.